### ISSUE TITLE
single quotation mark is not encoded url compatible

### ISSUE THREAD

**Issue by coyotede** [Mikhus/jsurl#14]
Even whitespaces are url-encoded... only the single quotation mark (') is not encoded url-compatible.

**Comment by Mikhus** [comment]
Hi, do you have any other details to make me understand the problem better and faster?

**Comment by longzheng** [comment]
Looks like it encodes single quotes properly

```javascript
new Url("http://hello world.com")
Object {protocol: "http", host: "hello%20world.com", port: "", path: "/", query: Object…}

new Url("http://hello'world.com")
Object {protocol: "http", host: "hello%27world.com", port: "", path: "/", query: Object…}
```

**Comment by Mikhus** [comment]
There was an issue on other url parts, fixed in 1.2.4
